pages/pipelines.md.erb

```diff
@@ -73,3 +73,21 @@ The display settings can be found in the `Pipeline Settings` section. Adding a d
 Clicking through to a pipeline page shows the [build history](#pipelines-page-walkthrough-build-history) for that pipeline, your starred branches, and the ten most recently built branches for that pipeline.
 
 <%= image "pipelines-detail.png", width: 1157/2, height: 731/2, alt: "Screenshot of the pipelines settings page" %>
+
+### Build page
+
+Clicking through to a build shows a build page with the full list of jobs and other steps in that build, the information about who triggered the build, and the controls for rebuilding or cancelling the build while it's in progress.
+
+On the build page, you can also view _All Builds_, _Edit Steps_ in the current build, open _Pipeline Settings_, or start a _New Build_.
+
+<%= image "inside-build-page.png", width: 2362/2, height: 1272/2, alt: "Inside the build page" %>
+
+Each job in a build has a footer that displays exit status information. It may include exit signal reason, which indicates that the Buildkite agent sent the signal because it either stopped or the job was cancelled.
```

Comment:
How about this?

```suggestion
Each job in a build has a footer that displays the job exit status. It may include the exit signal reason, which indicates whether the Buildkite agent stopped or the job was cancelled.
```